network/servicenetwork/servicenetwork_test.go

```diff
@@ -16,31 +16,14 @@
 
 package servicenetwork
 
-import (
-	"crypto"
-	"testing"
-
-	"github.com/insolar/insolar/certificate"
-	"github.com/insolar/insolar/component"
-	"github.com/insolar/insolar/core"
-	"github.com/insolar/insolar/core/delegationtoken"
-	"github.com/insolar/insolar/core/message"
-	"github.com/insolar/insolar/cryptography"
-	"github.com/insolar/insolar/messagebus"
-	"github.com/insolar/insolar/network"
-	"github.com/insolar/insolar/network/nodenetwork"
-	"github.com/insolar/insolar/platformpolicy"
-	"github.com/insolar/insolar/testutils"
-	"github.com/stretchr/testify/require"
-)
-
-func newTestNodeKeeper(nodeID core.RecordRef, address string, isBootstrap bool) network.NodeKeeper {
+/*
+func newTestNodeKeeper(nodeID core.RecordRef, address string, isBootstrap bool) (network.NodeKeeper, core.Node) {
```

Comment:
?

Reply:
Tests will not work until we use dependency injection there (there is separate task for this, if I am not mistaken)